app/views/layouts/devise_consolidated_login.html.erb

```diff
@@ -4,3 +4,4 @@
   body_class: 'login-page--consolidated',
   masthead_img_src: nil
 )%>
+<%= _link_to new_account_request_path, {text: 'No account? Register your bank here for free!', type: "secondary", size: "sm"}  %>
```

Comment:
@divagueame could you include this in the Bank specific login form? This consolidated login is used for both Bank and Partner users. Partners are invite-only and Banks can register via that link.